google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddRemarketingAction.java

```diff
@@ -0,0 +1,152 @@
+      return;
+    }
+
+    try {
+      new AddRemarketingAction().runExample(googleAdsClient, params.customerId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId) {
+
+    // Creates a remarketing action with the specified name.
+    RemarketingAction remarketingAction =
+        RemarketingAction.newBuilder()
+            .setName(StringValue.of("Remarketing action #" + System.currentTimeMillis()))
+            .build();
+
+    // Creates a remarketing action operation.
+    RemarketingActionOperation operation =
+        RemarketingActionOperation.newBuilder().setCreate(remarketingAction).build();
+
+    // Issues a mutate request to add the remarketing action and prints out some information.
+    String remarketingActionResourceName;
+    try (RemarketingActionServiceClient conversionActionServiceClient =
+        googleAdsClient.getLatestVersion().createRemarketingActionServiceClient()) {
+      MutateRemarketingActionsResponse response =
+          conversionActionServiceClient.mutateRemarketingActions(
+              Long.toString(customerId), Collections.singletonList(operation));
+      remarketingActionResourceName = response.getResults(0).getResourceName();
+      System.out.printf(
+          "Added remarketing action with resource name '%s'.%n", remarketingActionResourceName);
+    }
+
+    // Creates a query that retrieves the previously created remarketing action with its generated
+    // tag snippets.
+    String query =
+        String.format(
+            "SELECT remarketing_action.id, remarketing_action.name,"
+                + " remarketing_action.tag_snippets "
+                + "FROM remarketing_action "
+                + "WHERE remarketing_action.resource_name = '%s'",
+            remarketingActionResourceName);
+    try (GoogleAdsServiceClient googleAdsServiceClient =
+        googleAdsClient.getLatestVersion().createGoogleAdsServiceClient()) {
+      // Issues a search request.
+      SearchPagedResponse searchPagedResponse =
```

Comment:
You do not use a SearchGoogleAdsRequest with page size information for this search request, is it intended?

I am asking because the [golden](https://github.com/googleads/google-ads-php/blob/master/examples/Remarketing/AddRemarketingAction.php#L139) has it.

Reply:
Since we're only interested in one row from the result set, I don't see much value in creating a request object or setting the page size, and I think it hinders readability. Would prefer to leave this as-is unless you have strong objections.

Reply:
SGTM